assets/scss/components/stencil/paymentMethods/_paymentMethods.scss

```diff
@@ -125,6 +125,10 @@
     }
 }
 
```

Comment:
I think we need to increase the space between payment method sections (ex. between Credit Card section and Paypal section). I suggest changing `margin-top` to `2rem` instead of 0 in `.paymentMethodName` class.